lib/cdo/google/sheet.rb

```diff
@@ -21,29 +21,52 @@ def initialize(document_key)
       @document_key = document_key
     end
 
-    def export(sheet_name:, rows:)
+    def export(tab_name:, data:)
```

Comment:
I actually like the name `rows` better than `data`; it's (slightly) more specific. Is there a specific reason you changed it?

Reply:
No particular reason I can remember 😄  -- changed it back.